.github/workflows/vet_and_format.yml

```diff
@@ -0,0 +1,41 @@
+name: Vet and Format
+
+on: [push]
+      
+defaults:
+  run:
+    shell: bash
+    working-directory: ./
+
+jobs:
+  run-vet-gofmt:
```

Comment:
Just a thought, should we run vet and gofmt as two different jobs?
```
jobs:
  run-vet:
    ...
  run-go-fmt:
    ...
```

Reply:
Sure, let's do that.

Reply:
This makes all jobs run in parallel, which I suppose is better than them stopping if one of them fails, so let's do this.